addons/grenades/functions/fnc_incendiary.sqf

```diff
@@ -232,10 +234,7 @@ private _enginePosition = _vehicle modelToWorld (_vehicle selectionPosition _eng
 if (_position distance _enginePosition < EFFECT_SIZE * 2) then {
     _vehicle setHit [_engineSelection, 1];
 
-    if ("ace_cookoff" call EFUNC(common,isModLoaded)) then {
-        private _enabled = _vehicle getVariable [QEGVAR(cookoff,enable), EGVAR(cookoff,enable)];
-        if (_enabled in [2, true] || {_enabled isEqualTo 1 && {fullCrew [_vehicle, "", false] findIf {isPlayer (_x select 0)} != -1}}) then {
-            _vehicle call EFUNC(cookoff,engineFire);
-        };
+    if (["ace_cookoff"] call EFUNC(common,isModLoaded)) then {
```

Comment:
Can't we skip this check since we're just raising an event here?

Reply:
I'm not sure how it handles, but I've seen it in places in ACE. I thought it would be better practice to call the event only if the component was loaded.

Reply:
We'd just raise an eventhandler with no listener (well, 3rd party might listen I guess, is this API?)

Reply:
It is API in the docs.

Reply:
Then keep the check